src/scss/1_base/text.scss

```diff
@@ -165,7 +168,7 @@ strong {
 // Style guide: base.text.sub
 sup,
 sub {
-  @include adjust-font-size-to(10px);
+  font-size: 1rem;
```

Comment:
not sure why this rule was changed, but both evaluate to the same output

Reply:
I thought we were removing adjust-font-size-to declarations?